packages/docs/components/molecules/AnchorNav/js-api.md

```diff
@@ -0,0 +1,6 @@
+---
+title: AnchorNav JS API
+---
+
+# JS API
+The `AnchorNav` component extends the [`Base` class](https://js-toolkit.studiometa.dev/api/configuration.html) using the [WithTransition Decorator]. It inherits their respective APIs, so make sur have a look at them.
```

Comment:
**nitpick:** Unclear

All components are based on the `Base` class and the `AnchorNav` component does not use the `withTransition` decorator. 

I would suggest the following for this documentation page:

```
# JS API

## AnchorNav

The `AnchorNav` component does not expose any specific JavaScript API. 

## AnchorNavLink

The `AnchorNavLink` class implements the features of the [`Transition` primitive](/components/primitives/Transition/). 

## AnchorNavTarget

The `AnchorNavTarget` class implementes the features of the [`withMountWhenInView` decorator](https://js-toolkit.studiometa.dev/api/decorators/withMountWhenInView.html).
```